CRM/Utils/System/Drupal.php

```diff
@@ -265,6 +265,7 @@ public function addScript($code, $region) {
     return TRUE;
   }
 
+  
```

Comment:
Oh no! Jenkins not like whitespace! Whitespace not start with C.

Reply:
Cookie monster eat whitespace. Yum!!

Reply:
Please ensure you get consent before eating a cookie.

https://www.cookielaw.org/the-cookie-law/